inlang/source-code/website/package.json

```diff
@@ -16,10 +16,10 @@
 		"node": ">=18.0.0"
 	},
 	"dependencies": {
-		"@lix-js/server": "workspace:*",
 		"@isomorphic-git/cors-proxy": "^2.7.1",
 		"@lix-js/client": "workspace:*",
 		"@lix-js/fs": "workspace:*",
+		"@lix-js/server": "workspace:*",
```

Comment:
@flornkm this can't be. The website can't depend on the server when the server depends on the website.

Reply:
My mistake this is the lix server.

Reply:
@opral/lix well well this is even not a change. the dependency just changed from line 19 to 22. good example that semantic diffing is also desired for code